Remotion/Web/Development.WebTesting.IntegrationTests/Accessibility/AccessibilityViolationAnalysisTest.cs

```diff
@@ -118,9 +118,9 @@ public void AccessibilityButtonWithoutInnerHTML ()
               {
                   AccessibilityRequirementID.ButtonHasVisibleText,
                   AccessibilityRequirementID.AriaLabel,
-                  AccessibilityRequirementID.AriaLabeledBy,
-                  AccessibilityRequirementID.RolePresentation,
-                  AccessibilityRequirementID.RoleNone,
+                  AccessibilityRequirementID.AriaLabelledBy,
+                  //evtl roole presentaion durch presentational role ersetzen & role none
+                  AccessibilityRequirementID.PresentationalRole,
```

Comment:
Not sure about the desicion requested of me.

Reply:
Ah sorry, that was something I did for myself and forgot the remove again.